### ISSUE TITLE
Typo in code - system.src.js "transpier" should be "transpiler"

### ISSUE THREAD

**Issue by kudoz83** [systemjs/systemjs#1573]
Line 3668 of system.src.js:

```
 if (!transpiler.translate)
      throw new Error(loader.transpier + ' is not a valid transpiler plugin.');
```

should read loader.**transpiler** not loader._transpier_

Still working on why plugin-typescript isn't recognized as a valid transpiler... obviously the translate property isn't getting set.

**Comment by guybedford** [comment]
Thanks added in https://github.com/systemjs/systemjs/commit/eaa429242b733a837dc35d93aec0f768be5270c9.

**Comment by guybedford** [comment]
Released in 0.20.7.
